docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The date pickers are now part of the Advanced components
+description: XXXX
+date: 2022-04-03T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+---
+
+We're excited to announce that, after more than 18 month in the lab, the date picker components are now part of the X packages.
+
+## What is MUI X ?
+
+Since its creation, MUI have always kept at heart to provide great components for free.
+However, some _Advanced components_ such as date pickers require several full-time developers and can not be sustainable relying only on open-source income.
+To be able to keep improving the library and add _Advanced components_ without compromise on code quality, we decided in 2020 to launch MUI X, a new set of components available under two licenses:
+
+- MUI X: the MIT licensed, free to use version
+- MUI X Pro: the commercially licensed version with the most advanced features
+
+The Advanced component launched by our team have been the Data Grid, with the `@mui/x-data-grid` and `@mui/x-data-grid-pro` packages.
+More than one year later, this new source of income has allowed our company to hire several new developers and to increase the amount of work achieved, even on the open-source team.
+
+## Why move the date pickers to MUI X
+
+When the date pickers components have moved to the lab, it was already clear that some of their features would eventually be part of the commercially licensed packages.
+
+We just released two new packages:
+
+- `@mui/x-date-pickers` which contains all the MIT licensed components
+- `@mui/x-date-pickers-pro` which contains all the MIT and commercially licensed components
+
+> ⚠️The only components that we are moving to a commercial license are the date range components.
+> Those had a clear warning on the doc stating that they were intended for MUI X Pro.
+> All the rest of the components are available in `@mui/x-date-pickers` and will stay MIT and free forever.
```

Comment:
Instead of a warning here, I think this should be a section that explains which components are in the Pro package.

```suggestion
## Which components are only available in the Pro package?

The date range components can only be used with a commercial license.

All other date picker components are available in `mui/x-date-pickers` and will remain MIT licensed and free forever.
```